Gemfile.lock

```diff
@@ -405,69 +409,68 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
+      zstd-ruby (~> 1.5.5)
+    llhttp-ffi (0.5.0)
       ffi-compiler (~> 1.0)
       rake (~> 13.0)
-    lograge (0.12.0)
+    lograge (0.14.0)
       actionpack (>= 4)
       activesupport (>= 4)
       railties (>= 4)
       request_store (~> 1.0)
     logstash-event (1.2.02)
-    loofah (2.21.4)
+    loofah (2.22.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.12.0)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (1.0.2)
     marco-polo (2.0.1)
       railties (>= 6)
-    maxitest (3.3.0)
-      minitest (>= 5.0.0, < 5.12.0)
-    metaclass (0.0.4)
-    method_source (0.8.2)
-    mime-types (3.4.1)
+    maxitest (3.7.0)
+      minitest (>= 5.0.0, < 5.15.0)
+    method_source (1.0.0)
```

Comment:
### [method_source](http://banisterfiend.wordpress.com)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 0.8.2 → 1.0.0

[_~~change-log~~, [source-code](https://github.com/banister/method_source), [gem-diff](https://my.diffend.io/gems/method_source/0.8.2/1.0.0)_]





<details>
<summary>Commits</summary>

A change of **47** commits. See the full changes on [the compare page](https://github.com/banister/method_source/compare/v0.8.2...v1.0.0).

These are the first 10 commits:
- (3926a34) [Add Ruby 2 to Travis CI matrix. Eliminate warning.](https://github.com/banister/method_source/commit/3926a3405aad4b5c1dd76c35ef3a5e9c9bab4812)
- (5e5c556) [Merge pull request #24 from petergoldstein/feature/add_ruby_…](https://github.com/banister/method_source/commit/5e5c55642662c248e721282cc287b41a49778ee8)
- (8be4a97) [Add new Ruby 2.1 and 2.2 versions](https://github.com/banister/method_source/commit/8be4a97409fb91b20eab478bfc4a939757e73f62)
- (0c0a0ce) [Merge pull request #29 from nicolasleger/add_ruby_2_1_and_2_…](https://github.com/banister/method_source/commit/0c0a0cebbd11f2b44104b8c84b4f9952692422ad)
- (be10f27) [Add Travis CI status badge](https://github.com/banister/method_source/commit/be10f274ee02db4bd343ef29e2dc7ac4bf716e64)
- (05ce7fc) [Integer Unification for Ruby 2.4.0+](https://github.com/banister/method_source/commit/05ce7fcb22d3c7b0100d328a52376c6d5cb9272a)
- (3e2679d) [CI against Ruby 2.3](https://github.com/banister/method_source/commit/3e2679da63e4948fa87564c73700c6f4ab3264c0)
- (60e022e) [CI against Ruby 2.4.0](https://github.com/banister/method_source/commit/60e022edd866d19333f8825e87fdf5064ed73aee)
- (0e4ad87) [Merge pull request #39 from koic/integer_unification](https://github.com/banister/method_source/commit/0e4ad87b73794b271bffae4230382888207ed212)
- (6dcb116) [Merge pull request #40 from koic/use_latest_rubies](https://github.com/banister/method_source/commit/6dcb116e37e20e58f615ffe05a40bbe9a536e44a)


</details>